event_tracker.gemspec

```diff
@@ -34,5 +34,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'activerecord'
   spec.add_dependency 'activejob'
+  spec.add_dependency 'awesome_print'
   spec.add_dependency 'mixpanel-ruby', '~> 2.2', '>= 2.2.0'
+  spec.add_dependency 'require_all'
```

Comment:
Are we adding a 500LOC extra dependency to avoid writing something like:
```ruby
Dir[File.join('lib', '**', '*.rb')].each{ |lib| require lib }
```
or there's is additional advantages, future planned use that I am yet unaware of?

Reply:
+1